src/Resources/config/services.xml

```diff
@@ -21,7 +21,7 @@
 
         <service id="search.client" class="MeiliSearch\Client" public="true" lazy="true">
             <argument key="$url">%env(MEILISEARCH_URL)%</argument>
-            <argument key="$api_key">%env(MEILISEARCH_API_KEY)%</argument>
+            <argument key="$apiKey">%env(MEILISEARCH_API_KEY)%</argument>
```

Comment:
I'm worried... Why does this line fix the test?

Reply:
@curquiza Because in the package `meilisearch-php` the constructor argument has been rename from snake_case to camelCase since version `0.12.0`

Reply:
I did not realize that: for me the `apiKey` value was only internal... I did not know that user can access it! Good to know!

Reply:
So, in php, changing a variable name in the parameter for a method is breaking?

Reply:
@curquiza a constructor is `public` instead if you make it protected or private.
In Symfony I created a new `service` who create a new instance of `MeiliSearch/Client` and passing some parameters like the `apiKey`.
The code need to know the parameter's name to do that

Reply:
@emulienfou for me, you call the constructor this way:

```php
```

Reply:
@curquiza This is more related to Symfony than PHP itself.
You can find more information about Symfony Services right here: https://symfony.com/doc/current/service_container.html#manually-wiring-arguments